llvm/test/tools/sycl-post-link/spec-constants/SYCL-2020-zeroinitializer-array-of-arrays.ll

```diff
@@ -0,0 +1,115 @@
+; RUN: sycl-post-link --ir-output-only --spec-const=rt %s -S -o - | FileCheck %s --check-prefix CHECK
+; RUN: sycl-post-link --spec-const=rt %s -o %t.files.table | FileCheck %s -input-file=%t.files_0.prop --check-prefix CHECK-PROP
```

Comment:
```suggestion
; RUN: sycl-post-link --spec-const=rt %s -o %t.files.table
; RUN: FileCheck %s -input-file=%t.files_0.ll --check-prefix CHECK-IR
; RUN: FileCheck %s -input-file=%t.files_0.prop --check-prefix CHECK-PROP
```

Reply:
Applied the suggestion in 657aa24dcdc7f123e727a333c91c64900041ee4e. Thank you!